### PR TITLE
Add mavlink messages support.

### PR DESCRIPTION
**Pull request by douglaswsilva**
Hey guys, I'm implementing the reading mavlink messages from the SDK and was wondering if this is the best way to do it. Please let me know what you think. 

Note that I tried to use `DEBUG` in the enum, but the compiler complained, so I'm using `DEBUS = 7;` for now, let me know what would be a good alternative word.

### CODE REVIEW

File: protos/telemetry/telemetry.proto [hunk 1]
```diff
@@ -20,6 +20,7 @@ service TelemetryService {
     rpc SubscribeFlightMode(SubscribeFlightModeRequest) returns(stream FlightModeResponse) {}
     rpc SubscribeHealth(SubscribeHealthRequest) returns(stream HealthResponse) {}
     rpc SubscribeRcStatus(SubscribeRcStatusRequest) returns(stream RcStatusResponse) {}
+    rpc SubscribeMavMessage(SubscribeMavMessageRequest) returns(stream MavMessageResponse) {}
```
Comment (julianoes) [inline protos/telemetry/telemetry.proto]: I wouldn't call it `MavMessage`, what about `StatusText`?

File: protos/telemetry/telemetry.proto [hunk 2]
```diff
@@ -165,3 +171,20 @@ message RcStatus {
...
+    enum Type {
+        EMERGENCY = 0;
+        ALERT = 1;
+        CRITICAL = 2;
+        ERROR = 3;
+        WARNING = 4;
+        NOTICE = 5;
+        INFO = 6;
+        DEBUS = 7;
+        UNKNOWN = 8;
```
Comment (julianoes) [inline protos/telemetry/telemetry.proto]: The thing is that PX4 only ever uses `INFO`, `WARNING`, and `CRITICAL`, so we might as well drop the others.

Comment (JonasVautherin) [reply protos/telemetry/telemetry.proto]: Good to know. And we can bring the others back if needed later.

File: protos/telemetry/telemetry.proto (hunk 1 above)
Comment (douglaswsilva) [reply protos/telemetry/telemetry.proto]: Coll, will change the name. :)

File: protos/telemetry/telemetry.proto (hunk 2 above)
Comment (douglaswsilva) [reply protos/telemetry/telemetry.proto]: Ah, as Jonas said, good to know. I'll drop them.

**Pull request merged**